src/db/postgres/lrsql/postgres/record.clj

```diff
@@ -50,7 +50,7 @@
     (create-admin-account-table! tx)
     (create-credential-table! tx)
     (create-credential-to-scope-table! tx))
-  (-update-all! [_ tx]
+  (-update-all! [{{{jsonb? :enable-jsonb} :config} :tuning} tx]
```

Comment:
I believe that you can use `tuning` directly as a record field, since Component will add it upon start. Definitely try that to see if it works.

Reply:
```clojure
(defrecord PostgresBackedn [tuning]
  ...
  (-update-all! [_ tx]
    ...
    (if (-> tuning :config :enable-jsonb)
      (migrate-to-jsonb! tx)
      (migrate-to-json! tx)))
```

Reply:
word, that tracks, i'll see if it plays.

Reply:
https://github.com/yetanalytics/lrsql/pull/305/commits/f41d5a334efb5cdec3e929d06f54e2e72b5a6028